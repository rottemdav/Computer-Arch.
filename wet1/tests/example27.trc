32 4 0 2 local_history local_tables using_share_mid
0xbb248 T 0xfbe0
0x3250095c N 0x499d9c
0xeba80 N 0x934
0x1934 T 0xce898c
0xe6ec T 0xe7380
0xe81fa858 N 0xaaa3f4
0x8554f8 T 0x51614
0xcabaab44 N 0xe624
0xeba80 N 0x934
0x950 T 0xaf48c748
0xb30210 N 0xe7118
0x3dc378d8 N 0x568
0xecc7720 T 0x232cc
0x950 T 0xaf48c748
0x4fc T 0x2867258
0xd29ec N 0x798
0xbabd0 N 0x53ddc
0x178 N 0xe4c
0x53884 N 0xad331b54
0xe81fa858 N 0xaaa3f4
0xbb248 T 0xfbe0
0x4fc N 0x2867258
0x178 T 0xe4c
0xb30210 N 0xe7118
0xe6ec T 0xe7380
0x9c27fdc N 0x6368
0x53884 T 0xad331b54
0xd126a80 T 0x63c6f60c
0xbabd0 T 0x53ddc
0x676dc T 0x1fd140
0x065c N 0xb90
0xbabd0 T 0x53ddc
0xb1260 N 0xa3921e4
0xbb248 T 0xfbe0
0xe6ec T 0xe7380
0x1f8 T 0x9473ccc
0xd29ec N 0x798
0xbb248 T 0xfbe0
0x8554f8 N 0x51614
0xe81fa858 N 0xaaa3f4
0xf4b0 T 0x1f8
0x3dc378d8 N 0x568
0xf97b9b94 T 0xfe3d68
0xe6ec N 0xe7380
0xcabaab44 N 0xe624
0x5c28b8 T 0xbe4
0xcabaab44 T 0xe624
0xbabd0 N 0x53ddc
0xf4b0 N 0x1f8
0x950 T 0xaf48c748
0xd29ec T 0x798
0xd126a80 T 0x63c6f60c
0x1934 N 0xce898c
0x05230c T 0x5e4
0xecc7720 N 0x232cc
0x3250095c N 0x499d9c
0x74eca4 N 0xbf38bc
0x1f8 N 0x9473ccc
0x0a0 T 0x2ec34
0x5c28b8 T 0xbe4
0x1934 T 0xce898c
0xe6ec T 0xe7380
0xb1260 T 0xa3921e4
0x05230c N 0x5e4
0xe6ec T 0xe7380
0x5c28b8 T 0xbe4
0xcabaab44 N 0xe624
0x065c N 0xb90
0xf97b9b94 N 0xfe3d68
0xf97b9b94 N 0xfe3d68
0x950 T 0xaf48c748
0x87ea6e3c N 0xb6524
0xf97b9b94 T 0xfe3d68
0x676dc N 0x1fd140
0xb30210 T 0xe7118
0xf97b9b94 T 0xfe3d68
0xeba80 T 0x934
0xb1260 N 0xa3921e4
0x0a0 N 0x2ec34
0xb30210 T 0xe7118
0x4fc N 0x2867258
0x8554f8 N 0x51614
0xf4b0 N 0x1f8
0xcabaab44 N 0xe624
0xf370 T 0x9803d8
0x065c N 0xb90
0xd24 N 0x91f4
0x1934 N 0xce898c
0xf97b9b94 N 0xfe3d68
0xbb248 N 0xfbe0
0xeba80 N 0x934
0x2077c T 0xa488
0xcabaab44 N 0xe624
0x87ea6e3c N 0xb6524
0x87ea6e3c N 0xb6524
0xbabd0 T 0x53ddc
0x1f8 T 0x9473ccc
0x4fc T 0x2867258