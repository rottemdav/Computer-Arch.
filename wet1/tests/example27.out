0xbb248 N 0xbb24c
0x3250095c N 0x32500960
0xeba80 N 0xeba84
0x1934 N 0x1938
0xe6ec N 0xe6f0
0xe81fa858 N 0xe81fa85c
0x8554f8 N 0x8554fc
0xcabaab44 N 0xcabaab48
0xeba80 N 0xeba84
0x950 N 0x954
0xb30210 N 0xb30214
0x3dc378d8 N 0x3dc378dc
0xecc7720 N 0xecc7724
0x950 T 0xaf48c748
0x4fc N 0x500
0xd29ec T 0xe7380
0xbabd0 T 0xaf48c748
0x178 T 0x51614
0x53884 N 0x53888
0xe81fa858 N 0xe81fa85c
0xbb248 T 0xfbe0
0x4fc T 0x2867258
0x178 T 0xe4c
0xb30210 N 0xb30214
0xe6ec T 0x798
0x9c27fdc N 0x9c27fe0
0x53884 N 0x53888
0xd126a80 N 0xd126a84
0xbabd0 T 0x53ddc
0x676dc N 0x676e0
0x65c T 0x1fd140
0xbabd0 T 0x53ddc
0xb1260 N 0xb1264
0xbb248 T 0xfbe0
0xe6ec T 0xe7380
0x1f8 T 0xe4c
0xd29ec T 0xe7380
0xbb248 T 0xfbe0
0x8554f8 T 0x9473ccc
0xe81fa858 N 0xe81fa85c
0xf4b0 N 0xf4b4
0x3dc378d8 N 0x3dc378dc
0xf97b9b94 N 0xf97b9b98
0xe6ec T 0x798
0xcabaab44 N 0xcabaab48
0x5c28b8 N 0x5c28bc
0xcabaab44 N 0xcabaab48
0xbabd0 T 0x53ddc
0xf4b0 T 0x1f8
0x950 T 0x53ddc
0xd29ec T 0xe7380
0xd126a80 T 0x63c6f60c
0x1934 T 0xce898c
0x5230c N 0x52310
0xecc7720 T 0x232cc
0x3250095c T 0xb90
0x74eca4 N 0x74eca8
0x1f8 T 0x51614
0xa0 T 0x232cc
0x5c28b8 T 0xbe4
0x1934 T 0xce898c
0xe6ec T 0x798
0xb1260 N 0xb1264
0x5230c T 0x5e4
0xe6ec T 0xe7380
0x5c28b8 T 0xbe4
0xcabaab44 T 0xe624
0x65c T 0x499d9c
0xf97b9b94 T 0xfe3d68
0xf97b9b94 T 0xfe3d68
0x950 T 0xaf48c748
0x87ea6e3c N 0x87ea6e40
0xf97b9b94 T 0xfe3d68
0x676dc T 0xb90
0xb30210 N 0xb30214
0xf97b9b94 T 0xfe3d68
0xeba80 T 0x63c6f60c
0xb1260 T 0xa3921e4
0xa0 T 0x2ec34
0xb30210 T 0xe7118
0x4fc T 0x2867258
0x8554f8 T 0x9473ccc
0xf4b0 T 0x1f8
0xcabaab44 T 0xe624
0xf370 N 0xf374
0x65c N 0x660
0xd24 N 0xd28
0x1934 T 0xce898c
0xf97b9b94 T 0xfe3d68
0xbb248 T 0xfbe0
0xeba80 T 0x934
0x2077c T 0x2867258
0xcabaab44 T 0xe624
0x87ea6e3c N 0x87ea6e40
0x87ea6e3c N 0x87ea6e40
0xbabd0 N 0xbabd4
0x1f8 T 0x51614
0x4fc T 0xa488
flush_num: 60, br_num: 98, size: 2144b
